utils/logging.js

```diff
@@ -31,7 +31,7 @@ export const initLogCorrelation = (projectId) => {
  * https://cloud.google.com/run/docs/logging#special-fields
  */
 const formatters = {
-  level(label, number) {
+  level(label) {
```

Comment:
I think the link to winston is distracting on line 30 above, I had to dig to figure out what was going on here. 

https://getpino.io/#/docs/api?id=formatters-object might be a better link based on the configuration.

Reply:
Updated